src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -158,6 +177,29 @@ const InstancePage = (): JSX.Element => {
     }
   }, [bridge, bridgeError, getPageTitle, history, processorsError, t]);
 
+  useEffect(() => {
+    if (bridge?.error_handler?.type) {
```

Comment:
This is likely to need refactoring when @kostola merges https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1069

His PR adds a new [`type`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1069/files#diff-66557df441afc3933a43c346564ae30380002fb47b8a0f12612ebcf2e71ece79R307) that has no User configurable parameters and is effectively a marker to say "we handle it all internally".

I added `frontend` reviewers to his PR.. but you might like to create a new JIRA too.

Reply:
Created a new task also for the UI: https://issues.redhat.com/browse/MGDOBR-1056